11ty/definitions/colonialism.md

```diff
@@ -1,5 +1,29 @@
 ---
-title: Colonialism
+title: colonialism
 slug: colonialism
-defined: false
+speech: noun
+excerpt: the act of enforcing one's political and ideological dominance in a country or region that is not of one's origin
+sub_terms:
+    - text: modern
+    full_title: modern colonialism
+    - text: neo-
+    full_title: neo-colonialism
+    - text: settler
+    full_title: settler colonialism
+    - text: Western
+    full_title: Western colonialism
+defined: true
+reading:
+    - text: A brief context of colonialism
+      href: https://plato.stanford.edu/entries/colonialism/
 ---
+
+the act of forcing a group of people in a region or country to live under the policies and ideologies of an invading government or group of people for the purposes of asserting geo-political dominance.
+
+### Note
+
+Though [often associated with imperialism](https://plato.stanford.edu/entries/colonialism/), the two words have different Latin etymologies that drive how the connotation of colonialism and the word colony have evolved. The most notable instances of colonialism to the general public are Western colonialism and recently, neo-colonialism.
+
+The practice of colonialism was most prominent with the British, Ottoman, and other European empires upon regions in the Americas, Asia, and Africa where slave trade(s) was(were) implemented as an economic vehicle. Contrary to popular belief, the colonialism also affected people of European descent, most notably, the Irish. Some studies have tried to [clarify the question](https://www.jstor.org/stable/20720270?seq=1) of whether or not this is true, but they tend to be written by non-Native researchers, therefore muting a much needed context. Be mindful when researching this topic to note the origin of the researchers with respect to the colonies and colonisers that they are writing about.
+
+An extension of colonialism that appears in the 21st century is neo-colonialism - a more pervasive form of imposing ideological and economic dominance on a country after it has been declared independent from its colonizer. The stealth nature of neo-colonialism can be traced to treaties, agreements, and ideologies that have been imparted from the oppresors on the oppressed because of previous dependencies.
```

Comment:
You are always welcome to include new words! Let's go ahead and break it out please, and we can work on that definition together. 👍🏽

Reply:
Okie! Onward. Thanks Tatiana